forui/example/lib/sandbox.dart

```diff
@@ -71,7 +71,7 @@ class _SandboxState extends State<Sandbox> {
             ],
           ),
           const SizedBox(height: 20),
-          FButton(onPress: () {}, label: const Text('hi')),
+          FButton(prefix: const FButtonSpinner(), onPress: null, label: const Text('Hi')),
```

Comment:
Should we make the button in this example enabled? What do you think? @kawaijoe

Reply:
It should be disabled. I can't think of a usecase where the button should be enabled with the spinner.

Reply:
Fair enough.